### DISCUSSION TITLE
Can I use simple DFT (with atoms fixed) to train a model?

### DISCUSSION THREAD

**Discussion by TheoBardu** [mir-group/flare#402] [Q&A]
Hi to everybody. 
I'm  trying to study phonon transport properties of specific objects (in particular nanowires with different structures made by the same material, InAs). The classical approach is using phono3py, to generate supercells with atomic displacements to get the 3rd order force constants and so the lattice thermal conductivity and phonon relaxation times. 
Since one of my structure has not so much symmetries (and also many atoms in the unit cell), the DFT calculations needed are more than 10000. 
My idea were to use FLARE to train a ML model on a set of DFT calculations already done (for supercells created by phono3py with also the forces calculation) in a system which is pretty similar and then using the MLFF to calculate forces from the displacements generated by phono3py without go through thousands of DFT calculations.

Is there a way to do so in FLARE? 
Thank you so much.
Best,
Theo

**Comment by cjowen1** [comment]
Hi Theo,

Thank you for your message. Here is a tutorial from the Phoebe code, also developed within our group, where a FLARE MLFF can be used to determine force constants and thermal conductivity. 

https://phoebe.readthedocs.io/en/develop/tutorials/mlPhononTransport.html

-Cameron

**Reply by TheoBardu** [reply]
Hi Cameron,
Thank you so much for your answer. 
I actually saw that tutorial since I'm using Phoebe to calculate transport properties, but sincerely I'm quite confuse about it. It is not completely clear to me how I can generate the MLFF from all of my DFT outputs of quantum espresso. 
I mean, on Phoebe tutorial page is linked the [FLARE-LAMMPS tutorial](https://colab.research.google.com/drive/1qgGlfu1BlXQgSrnolS4c4AYeZ-2TaX5Y?usp=sharing#scrollTo=hlzfUR2XX06j) on collaborate and reading that I saw the way I can get the MLFF for offline training, but the input file used in that tutorial (called Al_otf_dft.xyz) is just one, and I suppose it's a snapshot of MD simulation performed previously. 
I don't have found anywhere if is mentioned how I can select all my output files from quantum espresso (where they are just DFT calculations of a supercell with some displacements) to be used as inputs for FLARE.

What I was thinking was to use all my DFT outputs to construct che MLFF, as if they were frames in a MD simulation, not just one output.
Is this is possible? 

Practically I have almost 500 DFT calculations performed on supercells created by Phono3Py. I would like to use this configurations and forces to train the ML model and then use the .flare as mentioned in the tutorial you linked to me.

I also read [this flare tutorial](https://colab.research.google.com/drive/1rZ-p3kN5CJbPJgD8HuQHSc7ecmwZYse6#scrollTo=2cTx8c62sATn) and at the point 4 for the offline training is mentioned that one can select more than 1 trajectory file in the otf section, but the files named aimd_trajectory1.xyz and aimd_trajectory2.xyz are not present in collaborate, so I can't look at them to reproduce them.

I apologize if this is obvious, but I'm new on ML applied to DFT calculation.

Thank you very much!
Theo
